### PR TITLE
Update Custom OBO Dashboard creation workflow

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

**Comment by anitacaron** [comment]
@matentzn, can you share the workflow image with me to update it, please?

**Comment by matentzn** [comment]
Its all in normal ODK!

File: docs/howto/deploy-custom-obo-dashboard.md [hunk 1]
```diff
@@ -29,6 +29,39 @@ We developed a completely automated variant of the Custom OBO Dashboard Workflow
      <img width="840" alt="image" src="https://user-images.githubusercontent.com/7070631/192295512-4ebf505c-c6e1-4448-9b22-735df8317eef.png">
  1. Click on `Visit site` and you should find your new shiny dashboard page!
 
+## Troubloushooting
+
+### Error on GitHub Action - Run dashboard section
+
+`Failed: make dashboard ROBOT_JAR=/tools/robot.jar ROBOT=robot  -B with return code 2`
+
+If there's only one ontology configured to run the dashboard, this may be causing the issue. Unfortunately, the dashboard expects at least three ontologies to complete the pipeline. We're working to fix this issue.
```
Comment (matentzn) [inline docs/howto/deploy-custom-obo-dashboard.md:38]:
```suggestion
There is a known bug at the moment requiring at least one ontology with a warning, error, info and pass, see https://github.com/OBOFoundry/OBO-Dashboard/issues/85. See issue for workaround. 
```

File: docs/howto/deploy-custom-obo-dashboard.md [hunk 2]
```diff
@@ -29,6 +29,39 @@ We developed a completely automated variant of the Custom OBO Dashboard Workflow
...
+
+1. In your `dashboard-config.yml`, add two other ontologies part of OBO Foundry, for example, Relation Ontology (RO) and Core Ontology for Biology and Biomedicine (COB).
+
+```yaml
+ontologies:
+  custom:
+    - id: ro
+    - id: cob
+```
```
Comment (matentzn) [inline docs/howto/deploy-custom-obo-dashboard.md:47]: Better explain the workaround using the TMP thing, just in case. This is certainly going to work.

Comment (anitacaron) [reply docs/howto/deploy-custom-obo-dashboard.md:47]: Yeah, thanks, I didn't update the docs 😬

Comment (anitacaron) [reply docs/howto/deploy-custom-obo-dashboard.md:47]: done

**Review by matentzn — APPROVED** [review]

**Pull request merged**